R/calculateDMM.R

```diff
@@ -261,7 +261,7 @@ setMethod("plotDMNFit", signature = c(x = "SummarizedExperiment"),
             geom_line() +
             theme_bw() +
             xlab("Number of Dirichlet Components") +
-            ylab(paste0("Model Fit (",type,")"))
+            xlab(paste0("Model Fit (",type,")"))
```

Comment:
Same comments here, let's use the `labs` function.

Reply:
done